packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/long-form-subnet.yaml

```diff
@@ -0,0 +1,37 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: SomeBucketName
+  Bucket2:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: 
+        Fn::GetAtt: [ Bucket1, BucketName ]
+      CorsConfiguration:
+        CorsRules:
+          - AllowedMethods:
+            - GET
+            - POST
+            AllowedOrigins:
+            - Fn::Join: ['', [{ Ref: Bucket1 }, '1']]
+            - origin2
+            MaxAge:
+              2
+  Subnet:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId:
+        Fn::If:
+        - AlwaysTrueCond
```

Comment:
This is wrong - there is no such condition here, so this will fail when we turn on this validaiton.

Maybe add that condition here?